### PR TITLE
Feat/more tests

### PR DESCRIPTION
**Pull request by stiebitzhofer**
# Description

* Add more certificates for validation tests against the JSON schema
* Allow using local translations for tests
* Add tests for translations
* Update PR template

Relates to https://github.com/orgs/s1seven/projects/3#card-65316011

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by getlarge — COMMENTED** [review]
If you're sure that the new invalid certificate will cover most error cases from users then i'll approve after the tests are running successfully.

File: test/fixtures/invalid_certificate_2.json [hunk 1]
```diff
@@ -0,0 +1,260 @@
+{
+  "RefSchemaUrl": "https://schemas.en10204.io/en10168-schemas/v0.0.3/schema.json",
+  "Certificate": {
+    "CertificateLanguages": ["DE"],
```
[the remark this answers is not in the document]
Comment (stiebitzhofer) [reply test/fixtures/invalid_certificate_2.json:4]: Good idea.

File: test/fixtures/invalid_certificate_2.json [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
...oRDFtjNMEwPs4cyWyWyBy3o0ZlGnwLjR3CnlaIPYoUKooGJrSyWVLBMcSDW2wIyC9/glw6s9YsxFBa2P0rdRqalLgOCWu3YN/6WRjcdSTu7drX49OCweuUojo5nDmSNvgxIzCixXSjd8vzapUbvVtetBhe9G6jL9o6w8FwSmmXw55WODyKFCqKBia0sllS0WK40bvlmZU4S505JrV7zMnhzHHnNHOdHPYce9rkDR47FxjOOiMYfL0ZTvrg7ZOU5xhDXdg9cOwwflj+K+7542v0OFoaD3a8FS3qNRGS8joA7wyTH2GX4mfsMR3KSQeLow1ZBcMiWBhCWQAzLxGG6BcSRuZpPmNrypJ9EptQ4XwAJWPVHjO63b+fsj8NRfIXQgF+kyXpT9wN6C6gUCZHMUscAkDAcRGQLASB/hkQF6w+L9iGZfC4DFlvscJ5z/AzWBeZDS8MP1l0+T56i58+s4XJri9bA9kayNZAJmtAW9mMoVK+4sQaMguD31zJxp63YbLx59031heHxdqS6Pjd2u2WFx2q5sGNlsdOfyHJYW9btOksq4t0CBaMJRhVtHrMavRhtDJDMNIBGpTlHzXDz2oDLluebA1kayBbA5mpgXTO8INeT7yaEgaGF/4/rSwMfV5IGFmh/y8kfYbRlqzQJ2HIEIYuwsCIvi0BDb5dOHPfc/AqMwIjeO0mpV0OsyTzYmHIYMfIKn2SmXLYdRBLD4aBZcfIWF3YazNbby85V3KYEqUvZm9HelCixXCjt+dF1qed0pTYXhIGhokeLEZu7GCEAfzC9va4ADtJnDkubJYsfpWCNahpNW6lUlMxYfCDXRScGGoNPnFdTE+MAE0JQOIjiFs7bKi2pBugsx0qkxp349byImFo3JEw9JMmUJ9EwtBl0oudckSW2a3EpxY3FkeeL4ZOENlkuJ2vKqoad1RPzFq5RRciS+Uz465yuGGIqkw+l5qtWTqpRQ4rhX8qiuo0MNfWOKyOo2K9HndujdoiikhYcgxIVwyd1KkLFUONG3BmhIo9ffjM7lq5CSEU4UWTNTGCSKU0Uo/6cfmVOxGdOZmBEaQOPxq/cm6ZH4213JqSmnHPlaXR1mHyRRPzkyEIlh+GXznX4UfjVx4Whl97/eTwK/fDD9KOMDCCyOlH41cepC1WDGtK8vMxiB3+f6vYXqlkMw4wAAAAAElFTkSuQmCC",
```
[the remark this answers is not in the document]
Comment (stiebitzhofer) [reply test/fixtures/invalid_certificate_2.json:17]: Will do.

File: test/fixtures/invalid_certificate_2.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (stiebitzhofer) [reply test/fixtures/invalid_certificate_2.json:4]: `valid_certificate_4.json` specifies polish. However, modifying the local Pl.json does not break things, shouldn't it?

File: test/fixtures/invalid_certificate_2.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (stiebitzhofer) [reply test/fixtures/invalid_certificate_2.json]: Reduced logo and CE sign size

File: test/fixtures/invalid_certificate_2.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (getlarge) [reply test/fixtures/invalid_certificate_2.json:4]: Yes it should, but local translations were not loaded until now.

**Comment by getlarge** [comment]
I fixed the path to the fixtures in the tests so now you can update the HTML and PDF expected results and check visually if they match your expectations.

File: test/render.spec.js [hunk 3]
```diff
@@ -68,10 +83,11 @@ describe('Render', function () {
         inputType: 'json',
         outputType: 'buffer',
         generatorPath,
+        translations,
```
Comment (getlarge) [inline test/render.spec.js:86]: Now we use local translations

File: test/validate.spec.js [hunk 4]
```diff
@@ -61,8 +70,177 @@ describe('Validate', function () {
...
+          params: { missingProperty: 'Z04' },
+          message: "must have required property 'Z04'",
+        },
+      ],
+    },
   ];
 
+  const languages = ['DE', 'EN', 'FR', 'PL'];
+  const translationProperties = {
+    certificateFields: [
```
Comment (getlarge) [inline test/validate.spec.js:152]: This is not a very smart way to test translations, but like mentione [here](https://github.com/s1seven/CoA-schemas/issues/9), at least that's one single source of truth of what properties the translations should contain.

**Review by getlarge — APPROVED** [review]

**Pull request merged**